### PR TITLE
Add speculation_watch_list dataset

### PR DESCRIPTION
**Pull request by kfinn**
https://github.com/nycdb/nycdb/issues/223

### CODE REVIEW

File: src/nycdb/datasets/speculation_watch_list.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    CapRate: numeric
+    BoroughCapRate: numeric
+    Yearqtr: text
+    Postcode: text
+    Latitude: numeric
+    Longitude: numeric
+    CommunityBoard: text
+    CouncilDistrict: text
+    CensusTract: text
+    BIN: integer
```
Comment (austensen) [inline src/nycdb/datasets/speculation_watch_list.yml:29]:
```suggestion
    BIN: char(7)
```

**Review by austensen — COMMENTED** [review]
Looks good! I think you just need to add the new entry for [verify](https://github.com/nycdb/nycdb/blob/main/src/nycdb/verify.py) and it's good to go!

**Comment by kfinn** [comment]
Just added the verify step!

File: README.md [hunk 2]
```diff
@@ -29,6 +29,7 @@ Residents, lawyers, tenants, and organizers who want to use data in their strugg
 - J-51 Exemptions
 - [OCA Housing Court Records](https://github.com/nycdb/nycdb/wiki/Dataset:-OCA-Housing-Court-Records) (zipcode)
 - [BBLs of NYC Housing Authority (NYCHA) Developments](https://github.com/nycdb/nycdb/wiki/Dataset:-NYCHA-BBLs) — From NYCHA via [JustFix's scraper tool](https://github.com/JustFixNYC/nycha-scraper)
+- [Speculation Watch List](https://data.cityofnewyork.us/Housing-Development/Speculation-Watch-List/adax-9mit)
```
Comment (austensen) [inline README.md:32]:
```suggestion
- [Speculation Watch List](https://github.com/nycdb/nycdb/wiki/Dataset:-Speculation-Watch-List)
```

**Review by austensen — APPROVED** [review]
Looks good! I thought it might make sense to just create the wiki page now so we can link it from the readme while we're at it, then it can just get filled out later without another PR needed.

**Pull request merged**